src/SignalR/clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -370,6 +370,10 @@ export class HttpConnection implements IConnection {
                     await this.transport!.connect(connectUrl, requestedTransferFormat);
                     return;
                 } catch (ex) {
+                    if (!ex) {
```

Comment:
It might be better to do this in the WebSocket transport and avoid throwing `null` altogether so that it can throw a more helpful unhelpful message :). We could also just do both (in case the transport does indeed throw `null`).